src/js/components/SharedComponents/CalendarRangeDatePicker.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { PropTypes } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+
+import Moment from 'moment';
+
+function Navbar({
+  onPreviousClick,
+  onNextClick,
+  className
+}) {
+  const styleLeft = {
+    position: 'relative'
+  };
+  const styleRight = {
+    position: 'relative',
+    marginLeft: 'auto'
+  };
+  return (
+      <div className={className}>
+          <button style={styleLeft} onClick={() => onPreviousClick()}>
```

Comment:
Could you put each of the properties on a new line? Also, is there a reason `onClick={onPreviousClick}` doesn't work?